flexget/plugins/urlrewrite/newpct.py

```diff
@@ -24,26 +24,42 @@ class UrlRewriteNewPCT(object):
     def url_rewritable(self, task, entry):
         url = entry['url']
         rewritable_regex = '^http:\/\/(www.)?newpct1?.com\/.*'
-        return re.match(rewritable_regex, url) and not url.startswith('http://www.newpct.com/descargar/')
+        return re.match(rewritable_regex, url) and not url.endswith('.torrent')
 
     # urlrewriter API
     def url_rewrite(self, task, entry):
         entry['url'] = self.parse_download_page(entry['url'])
 
     @plugin.internet(log)
     def parse_download_page(self, url):
+        if 'newpct1' in url:
+            log.verbose('Newpct1 URL: %s', url)
+            url = url.replace('newpct1.com/', 'newpct1.com/descarga-torrent/')
+        else:
+            log.verbose('Newpct URL: %s', url)
+
         page = requests.get(url)
```

Comment:
It should never fail because first I check the string "newpct1" so it will always contain "newpct1.com" but if you want I can change it to "if 'newpct1.com' in url:"